admiral/pkg/controller/common/types.go

```diff
@@ -137,13 +137,13 @@ func (s *SidecarEgressMap) Put(asset string, namespace string, fqdn string) {
 	s.mutex.Lock()
 	var mapVal = s.cache[asset]
 	if mapVal == nil {
-		mapVal = make([]SidecarEgress, 0)
```

Comment:
Can we change the variable name from `asset` to `identity`?

Reply:
Done